docs/modules/ROOT/pages/autotasks.adoc

```diff
@@ -194,12 +194,12 @@ Autotasks are executed in a https://nodejs.org/dist/latest-v12.x/docs/api/[node
 "@gnosis.pm/safe-ethers-adapters": "^0.1.0-alpha.3",
 "axios": "0.21.2",
 "axios-retry": "3.1.9",
-"defender-admin-client": "1.25.0",
-"defender-autotask-client": "1.25.0",
+"defender-admin-client": "1.28.0",
```

Comment:
@zeljkoX please make sure this is accurate for next release and add info about choosing dependencies

Reply:
@mok0230 Just updated.